OpenRA.Game/Settings.cs

```diff
@@ -39,7 +39,22 @@ public class ServerSettings
 		public string Name = "OpenRA Game";
 
 		[Desc("Sets the internal port.")]
-		public int ListenPort = 1234;
+		private int listenPort = 49152;
+		public int ListenPort
+		{
+			get
+			{
+				return this.listenPort;
+			}
+
+			set
```

Comment:
Please move the validation logic into `ServerCreationLogic`. These settings classes are automatically synced with files on disk, so shouldn't contain any logic that isn't related to this purpose.